config/config.nonprod.yaml

```diff
@@ -12,9 +12,24 @@
   contact: dtownsend@mozilla.com
   description: Flowstate whiteboard tag
   enabled: true
-  module: jbi.actions.default_with_assignee_and_status
   parameters:
     jira_project_key: MR2
+    steps:
+      new:
+        - create_issue
+        - maybe_delete_duplicate
+        - add_link_to_bugzilla
+        - add_link_to_jira
+        - maybe_assign_jira_user
+        - maybe_update_issue_resolution
+        - maybe_update_issue_status
+      existing:
+        - update_issue
+        - maybe_assign_jira_user
+        - maybe_update_issue_resolution
+        - maybe_update_issue_status
+      comment:
+        - create_comment
```

Comment:
Is there a way to set up the configuration where we only need to override the step section that deviates from default? Like in this "custom" workflow and the default workflow, the comment steps remain the same, so it might be nice to not have to specify that.

Reply:
Do you mean like a merge of default steps and the configuration ones?

For example, if the `comment` group is not specified we use the default steps for `comment`. If it is specified we follow the configuration (eg. specify it and leave it empty to remove the default behaviour)

Reply:
I did the merge, let me know if that's what you had in mind

Reply:
Yep, that's what I was talking about! Nice ✅